### ISSUE TITLE
Reorganizing

### ISSUE THREAD

**Issue by marwahaha** [rachelslaybaugh/NE155#24]
It would be great to make a img/ folder within each lecture grouping. That way, it's easier to find the pdf, tex, or ipynb versions of notes (since the images are supplementary).

An example directory where it would help: https://github.com/rachelslaybaugh/NE155/tree/gh-pages/02-computers
Let me know what you think :-)

**Comment by rachelslaybaugh** [comment]
Makes sense. I might not do the reorg until spring--but this is a great suggestion!
